R/call_derivation.R

```diff
@@ -111,10 +111,10 @@ call_derivation <- function(dataset = NULL, derivation, variable_params, ...) {
 
   fixed_params <- eval(substitute(alist(...)))
   if (length(fixed_params) == 0L) {
-    abort("At least one argument must be set inside `...`")
+    cli_abort("At least one argument must be set inside {.arg ...}.")
```

Comment:
Do we want a snapshot for this abort - i don't see if in any of our tests or snapshots?